eZ/Publish/Core/Persistence/Legacy/Content/Language/MaskGenerator.php

```diff
@@ -48,6 +49,11 @@ public function generateLanguageMask(array $languages)
         }
 
         foreach ($languages as $language => $value) {
+            if (is_int($language)) {
```

Comment:
A need for that scares me :)
But yeah, let's keep it.

Reply:
I added it mostly for debugging as I overlooked that it expects flipped array.

And I'm not sure, but when debugging it I seem to have gotten clues that some existing calls use it wrong, but as there does not seem to be any failures on this it's either silenced by other expected exceptions, or it was the code I was working on that tricked me.

Anyway, I'm planning of changing this to batch load in another PR, and then this logic will be gone, so for me we can actually skip it here.

The reason why it expects flipped array is that it's primarly meant for translated name arrays which comes in such format.

So ideally we should rename and add one for normal `generateLanguageMask(array $languageCodes, bool $useAlwaysAvailable)`

Reply:
Removed renaming of this in this PR, that is more for master.